src/cmdlib.sh

```diff
@@ -174,7 +174,21 @@ prepare_build() {
     mkdir "${tmp_builddir}"
 
     configdir=${workdir}/src/config
-    manifest=${configdir}/manifest.yaml
+
+    init_config="${workdir}/src/config.json"
+    if [[ -f "${init_config}" ]]; then
+        variant="$(jq --raw-output '."coreos-assembler.config-variant"' "${init_config}")"
+        osver="$(head -1 "${workdir}/src/config/variant_${variant}")"
+        manifest="${configdir}/manifest-${osver}.yaml"
+        image="${configdir}/image-${osver}.yaml"
```

Comment:
Wasn't the idea with this latest iteration that we'd get rid of the `variant_${variant}` files?

Reply:
It was. I missed those indeed.